### ISSUE TITLE
It's possible to land on the toolbar container in a custom tab when using a screen reader [Bug]

### ISSUE THREAD

**Issue by DiegoARL38** [mozilla-mobile/fenix#3198]
## Steps to reproduce
1. Open Fenix with a screen reader on.
1. Go to Settings, Privacy notice.
1. Explore via touch the screen.
    1. Expected behavior
The toolbar container is not focusable.
    1. Actual behavior
It's possible to land in the toolbar container.
    1. Device information

* Android device: TP-Link Neffos C5 Max (Android 5.1)
* Fenix version: Build #11561807
    1. Notes
* Tested with both TalkBack and ShinePlus.

**Comment by kbrosnan** [comment]
This is expected behavior. The view is a custom tab which do not allow the user to interact with the address.

**Comment by kbrosnan** [comment]
@DiegoARL38 would you clarify the behavior for me. The screen reader should be able to read the adderss URL but not focus the address area. This would be expected. If the screen reader cannot read the address for a custom tab then that would be a serious bug.

**Comment by DiegoARL38** [comment]
@kbrosnan It's possible to land on both the page title and the domain. It's also possible to land on the toolbar container itself, which is what i find odd...
